app/models/experimentation/notifications_experiment.rb

```diff
@@ -47,6 +50,7 @@ def self.eligible_patients
                                              .where(status: :scheduled)
                                              .group(:patient_id)
                                              .having("count(patient_id) > 1"))
+        .where.not(block_region: {id: EXCLUDED_BLOCKS[CountryConfig.current[:abbreviation]]})
```

Comment:
Maybe make this conditional on having excluded blocks for the current deployment?

I'm thinking if we ever run experiments in a country without excluded blocks, I think it will look for block_regions where the id is not null.  That won't change the query results I think, but it does make the query more complex ( and slower).